.github/workflows/update-testapp.yml

```diff
@@ -0,0 +1,24 @@
+name: Update Assurance Test App APK
+
+on: workflow_dispatch
+
+jobs:
+  update-testapp:
+    runs-on: macos-latest
+
+    steps:
+    
+    - name: Checkout
+      uses: actions/checkout@v3.1.0
+      with:
+        ref: dev
+
+    - name: Generate AssuranceTestApp APK
+      run: make build-app
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v5
+      with:
+        commit-message: Update Assurance Test App APK
+        title: Update AssuranceTestApp APK
+        base: dev
```

Comment:
Additionally, the android release workflow gets auto triggered on push to main so. I made the PR to be raised from `dev` so that we could follow the below workflow to keep the release flow linear: 
- make changes to test app in dev branch and commit them
- run the workflow to generate the test apk with the changes in dev
-  merge this PR to dev
- propagate this apk to `stage` and `main` via trivial merges

My thinking was that if we were to make the pr against `main` then we would need to reverse merge it to `dev` to keep in sync and also change the release workflow trigger to be manual.